### PR TITLE
Update DCU view refresh job to run in PSQL container

### PR DESCRIPTION
**Pull request by SWhitlam**

### CODE REVIEW

**Comment by nickmcmahon01** [comment]
Because this is a Major, the reference in audit/extracts drone.yml will also need updating, or it won't get deployed to Dev, only QA and beyond. I think Major is desirable here because we can control the promotion of it into prod.

**Comment by LiamMacP** [comment]
We need to test this against the hocs-extracts read-only replica - I'm not convinced this would work there. Due to potential performance implications - we should likely run this later in the afternoon.

**Comment by LiamMacP** [comment]
Just to confirm this needs to be run against the `hocs-audit`.

`ERROR:  cannot execute REFRESH MATERIALIZED VIEW in a read-only transaction`

File: charts/hocs-audit/values.yaml [hunk 1]
```diff
@@ -23,3 +23,5 @@ hocs-generic-service:
 
 dcuCaseView:
   enabled: false
+  refresh: '30 5 * * *'
```
Comment (nickmcmahon01) [inline charts/hocs-audit/values.yaml:26]: I'm injecting a requirement at PR but can this be 2:15?

Comment (nickmcmahon01) [inline charts/hocs-audit/values.yaml:26]:
```suggestion
  refresh: '15 2 * * *'
```

File: charts/hocs-audit/Chart.yaml [hunk 2]
```diff
@@ -3,5 +3,5 @@ name: hocs-audit
 version: 4.1.0
```
Comment (nickmcmahon01) [inline charts/hocs-audit/Chart.yaml:3]:
```suggestion
version: 5.0.0
```

File: charts/hocs-audit/Chart.yaml [hunk 3]
```diff
@@ -3,5 +3,5 @@ name: hocs-audit
...
 dependencies:
   - name: hocs-generic-service
-    version: ^5.0.0
+    version: ^6.0.0
```
Comment (nickmcmahon01) [inline charts/hocs-audit/Chart.yaml:6]:
```suggestion
    version: ^5.0.0
```

**Pull request merged**